app/controllers/home_controller.rb

```diff
@@ -1,3 +1,11 @@
 class WeekFourAssessmentApp
-  # Your code will go here.
+  get '/' do
+    @name = 'Francesco'
+    erb :welcome
+  end
+
+  get '/name/:name' do
+    User.create(name: params[:name])
```

Comment:
I can's see that you are rendering a erb in this `get` route. Why not?

Reply:
In point 5 of the README file it is requested for the second test to be passed into a view, but for the other test it is not requested to pass it into a view so it was unnecessary.